openmdao/docs/features/recording/getting_derivatives.rst

```diff
@@ -0,0 +1,14 @@
+****************************************
+Getting Derivatives with the Case Reader
+****************************************
+
+If you are running with a driver that computes derivatives, you can also record and read them through the `CaseReader`.
```

Comment:
This may be a little picky, but this sentence makes it sound like you record them through the CaseReader.
Maybe "you can also record derivatives, and then subsequently read them through the `CaseReader`.  ?  Read it over and see if you agree. Maybe only need to change it if there are other changes needed to the PR.

Reply:
I agree with Keith's comment that this could use a bit of wordsmithing

Reply:
Reworded this.